src/RollbarSourceMapPlugin.js

```diff
@@ -93,12 +95,22 @@ class RollbarSourceMapPlugin {
       }
     });
 
+    // transform sourceFile and sourceMap names
+    let transformResults;
+    if (this.transform) {
```

Comment:
Might be good to check if transform is a function `if (typeof this.transform === 'function')`.